modules/fedimint-lnv2-tests/src/bin/tests.rs

```diff
@@ -81,15 +81,15 @@ async fn test_self_payment(dev_fed: &DevJitFed) -> anyhow::Result<()> {
 
 async fn fetch_invoice(
     client: &Client,
-    gw_address: &String,
+    gw_address: &str,
 ) -> anyhow::Result<(Bolt11Invoice, OperationId)> {
     Ok(serde_json::from_value::<(Bolt11Invoice, OperationId)>(
         cmd!(
             client,
             "module",
             "lnv2",
             "receive",
-            gw_address.clone(),
+            gw_address.to_owned(),
```

Comment:
Nit: Do we even need `.to_owned()`? I think we can just put `gw_address`

Reply:
Right.